exist-core/src/test/java/org/exist/xquery/OpNumericTest.java

```diff
@@ -280,4 +283,63 @@ public void minusReturnType9() {
 	public void minusReturnType10() {
 		assertEquals(Type.TIME, buildOp(ArithmeticOperator.SUBTRACTION, time, dtDuration).returnsType());
 	}
+
```

Comment:
@adamretter that would be a good candidate to be turned into an JUnit 5 parameterized test in the future...